### PR TITLE
Add version 10 highlights to release notes

### LINKED ISSUE #7770: Create release notes for version 10.0.0
## Changes to the Manage experiments tool
The Manage experiments tool has become nicer to work with. Before you would just get text information about each experiment:
Screenshot from 2024-04-30 08-49-37

Now there is a separate panel for viewing the information:

image

## Longer retries for license error

For ECL100 jobs there is a possibility that the license server may be overloaded by too many concurrent realizations. To mitigate this we have increased the retrying mechanism for checking the license error from max 6 minutes to between 19 and 32 minutes (depending on randomized values). In order to check whether your realization is waiting for the license server to respond, while running the experiment, click on "show details":
screenshot_from_2024-04-29_15-57-24_720

Click on the square for the long running realization, 0 in the image:
screenshot_from_2024-04-29_15-57-37_720

Now click the "OPEN" button in the column STDERR for row ECLIPSE100:

screenshot_from_2024-04-29_15-57-45_720

If the message contains "Eclipse failed due to license failure, retrying in XXs" then the license server is busy and we will automatically retry running eclipse in the specified number of seconds.

screenshot_from_2024-04-29_15-57-51_720

## Breaking changes for plugins, forward models and api endpoints

There are a few breaking changes, which only effects users of the storage api and plugins and ertscripts that uses LibresFacade:

* The storage api endpoint "/ensembles/{ensemble_id}/responses/{response_name}/data" is removed.
* The deprecated methods `grid`, `gen_data_keys`, `gen_kw_keys`, `all_data_type_keys`, `observation_keys` of `LibresFacade` are removed.

List of new commits in 10.0.0
### BREAKING CHANGES
aea1d7cb0ac8fe07102f910c756924174ab41904 Remove storage API endpoint "/ensembles/{ensemble_id}/responses/{response_name}/data" by Øyvind Eide (#7566)
b0905379911924f0a08d96890ea50b9d47057439 Remove deprecated method LibresFacade.grid by Eivind Jahren (#7567)
c656cb2f372788f70b2157ec324958229ba9666e Remove method `gen_data_keys` from LibresFacade by Øyvind Eide (#7520)
4b0382af7915e0dd4d2bff865e6559c5ca7c23da Remove method `gen_kw_keys` from LibresFacade by Øyvind Eide (#7520)
be986e75b55c9566da3a1a01c21c976344abfe50 Remove method `all_data_type_keys` from LibresFacade by Øyvind Eide (#7520)
7812d6cf698300eaadaae8e46719dc8b2817ae3b Remove method `observation_keys` from LibresFacade by Øyvind Eide (#7520)

### GUI CHANGES
c86ab2db125d49f7e40d4c22413e1d534b5f3e66 Write reponse as yaml by Frode Aarstad (#7692)
1065153f210cd65623165f62c0de0c39493a2ab5 Rename Jobs to Forward Model in GUI by Jonathan Karlsen (#7652)
82db76d96cbb5f43ab20c2c567ba88e730bee1be Remove duplicate data by Frode Aarstad (#7623)
01c3c3090576fb6657de818c5fe059e84d231319 Remove resdata version from about page by Eivind Jahren (#7460)
53daee4bb903aecd49c32870e3138b2b0cd2234e Add an experiment info widget by Frode Aarstad (#7525)

### GUI BUG FIXES
2ba514358dbb59f71a36c40b85b085bf973cbcf6 Make sure that rerun jobs do not show an error message in the GUI by Dan Sava (#7696)
fc699f6df4148f8194f814892a3b1cb512879f63 Check for selected ensemble in evaluate ensemble panel by Frode Aarstad (#7680)
bb9b46bd31a9df05afe7fb12312454b98fe34828 Fix GUI RunDialog not expanding details header by Jonathan Karlsen (#7617)
e5aa0f340607c4266ed34f2d84d1389c7b9386dc Fix memory being reported without units in GUI by Jonathan Karlsen (#7656)

### BUG FIXES
82a2d12f48ae7036823c357a45c5ff5b9d4e776d Fixed an issue where local queue would error due to it being killed before starting by Håvard Berland (#7710)
10601d437f91918e52ad1c295e4da98f90aa6a18 Ensure that opening an older version of ert with a newer version of storage shows an informative error message by Frode Aarstad  (#7719)
8336e5142db31c940a6abc0e9e775f7c0fdd96fa Fixed formatting of file open error on bsub (LSF) in by Håvard Berland (#7671)
bd0d082de0b177b6f18460b0a9e2a3cb57355be6 Ensure that the storage lock is aquired before writing by Øyvind Eide (#7570)
fc8a4055da65fc6521b70b5300c2a87ecfc76f74 Fixed an issue where empty storage directories were migrated by Øyvind Eide (#7570)
4ababea024d3d136039bf627212b8433c726be54 Establish connection and empty the event queue before cancelling tasks by Julius Parulek (#7562)
d441394bfef89a5b62b16dbc86c3e77e74221d2f Fix bug where all ensembles would show instead of just the initialized ones by Øyvind Eide (#7538)

### DOCUMENTATION
907f0b8f2f628416edfbe66ef94ffd3d86ed4ab7 Fix documentation for LSF memory booking by Jonathan Karlsen (#7654)
2600d207bd3d77507ef78340a0b62c2f7cbc851d Document storage by Aron Høyer (#7254)
48a15b75a49e3a8c8a0690cf87e4a1344b766611 Document how delete_directory works on symlinks by Håvard Berland (#7444)
3fdff884848078484828219bf7623f77294515a2 Split release notes in highlighted changes and change log by Øyvind Eide (#7542)

### SCHEDULER LSFDRIVER
606a4e0df82195f6feb89f40d0d731a6373a96dc Have scheduler bkill not retry on error `Job already finished` by Jonathan Karlsen (#7714)
0217bfdc819e9141a9a7f84d487cf032a9cf29ca Add -o option to bsub for stdout from LSF by Håvard Berland (#7724)
8548bcd685e05bed0b5b6464fca5595c48ce6bea Retry bkill on intermittent ssh failures by Håvard Berland 
120de548ff20f21eb0a5d7f1a658b5b0bb7a34f8 Let jobs failing in LSF be resubmitted by Håvard Berland (#7684)
4e7300ec70b2b83ddcdea7268a33fe7da7bd69bd Fix LSF driver logging wrong message when killing by Jonathan Karlsen (#7672)
8082875060e07e8a9e64bad774bc7300f88349f9 Implement EXCLUDE_HOST for scheduler lsf driver by Jonathan Karlsen (#7543)
70122c96c38d5ac4db324de14ca3f23c9ec9dd05 Solve race condition in lsf_driver for job_ids by Håvard Berland (#7581)
ade18beaece581e82cdc60c323a35bc3d7a0ec85 Revert "Implement EXCLUDE_HOST for scheduler lsf driver" by Håvard Berland  (#7547)
5bb33aa3df473dd25659fedf9b6e14c1b622dbed Implement EXCLUDE_HOST for scheduler lsf driver by Jonathan Karlsen (#7441)
89f5179a30dca86bc50cc3ad5df906e8e6c2b2a0 Implement LSF_RESOURCE queue option for LSF_DRIVER by Jonathan Karlsen (#7441)
1c1badd7671e068d83d17885c151455065ec85d2 Make fallback mechanism for failing bjobs by Håvard Berland (#7299)

### REFACTORING
108a888514e3edd8ad0f18c853348860edf2aba2 Remove unused code ensemble_evaluator_utils by Jonathan Karlsen (#7682)
e4d8f7f3902013976f54f324e1bf9dd6d08363c4 Rename JobRunner to ForwardModelRunner by Jonathan Karlsen (#7628)
29d0fe49a2403af01ada62e271aab04927631ede Move duplicate ev_types to ert/event_type_constants.py by Jonathan Karlsen (#7683)
1eaed7c83af9159c693f07554bb688d21d99f7ed Have async functions use async_utils eventloop by Jonathan Karlsen (#7624)
c600efbbe3b15f441ffdbaae86a2201d1e270828 Remove start sync event from scheduler job.__call__ by Jonathan Karlsen (#7616)
d55e0ef9870dc95aefd8cdaeaccc12c044391b26 Rename async_utils get_event_loop() to get_running_loop() by Jonathan Karlsen (#7618)
20768faedd3de45a7e1f73b609de28884dbbd3ac Fix py3.12 asyncio warning in async_utils by Jonathan Karlsen (#7597)
bc992ec461a612af8c31fe630cad4887f41fc3a7 Rename Job.__call__ to job.run in scheduler by Jonathan Karlsen (#7615)
afd2437d33cb404892141b35c14193a350599b09 Remove pass statement by Håvard Berland (#7607)
b4dc96d54f3c2eaa0a8a60f539545887599182e6 Solve py312 warning on pkgutil.get_loader (#7597)
7ba3fb551d7f18fb43b8a4e9500b28a72d364f7c Solve deprecation warning from datetime on UTC by Håvard Berland (#7584)
118697112ec5893510d377441e89e5fb52b3cc6f Refactor _get_obs_and_measure_data by Feda Curic (#7480)
83b3dd2a2345bb2eafadf53cf54b276cd969b9bf Refactor storage migrations by Øyvind Eide (#7570)
b21fab1a816b7bc64d88006e122c3d5f443e7a0a Use ruff also as formatter by Lars Evje (#7503)
b71fdde61f2c3ad7b36bb6a7a5cae8708ba7beb2 Fix ErtPlugin interface staticmethods Jonathan Karlsen (#7540)
15ff24633c25cc5b93ec6b9343c09651f468e001 Fix ruff preview rule PLR6301 by Jonathan Karlsen (#7529)
85fd3cd47501fd888e8448b279f164e3e7c4794e Move ignore check one level higher by Øyvind Eide (#7570)

### NON-USERFACING CHANGES
581745044c88f2708bc26136667edf4d9b2a2adf Improve scheduler driver execute_with_retries method to log exit code, stdout, and stderr by Håvard Berland (#7769)
0eb422f5bcd6fa63ecc10d5fff09d812515fc0ed Log stdout when bhist fails by Håvard Berland (#7697)
b9b60b300fa8429063186ffdd563bff2d94194e7 Changed how PARALLEL is read from DATAFILE by Eivind Jahren (#7460)
0e3f9b5ad606814d1098f0092e1efbfd71cefbaa Use macOS-14 runners for ert by Andreas Eknes (#7592)
5428155b98bf2878eb9d5148633a74a54c295e43 Make resdata a dev-dependency by Eivind Jahren (#7460)
5f167f4ffbc73c21bc305aefdb278d44cfd20ccb Remove logging of util_abort, it can no longer happen by Eivind Jahren (#7460)
3be64fd3bc375df9686ef5676d2fa919226ebb01 Add logging to LocalDriver by Håvard Berland (#7549)
3bc1f3050dc61430a8a1fbb7826e6eae39c5981a Have lsf driver bkill with SIGKILL after SIGTERM by Jonathan Karlsen (#7482)
a71a6dfa90f622408c349eb6edf9e415d2d48f22 Have lsf_driver specify SIGKILL when bkilling by Jonathan Karlsen (#7433)
66e7fe541923f354544e0dc3388c6388171967a4 Improved log message when killing jobs on local queue by Håvard Berland (#7556)

### TESTING
4ec192af30000cf49ba253f81c7dddd2c7091dc2 Add SIGKILL to possible returncodes from kill by Håvard Berland (#7586)
b8855795bb3dfa571ef0f42d1aa32fdaebbd352b Fix dark storage performance benchmark test not compatible with async_utils event loop by Jonathan Karlsen (#7737)
998751b373fe45d5c660e45f6c2862de280260e7 Increase tolerance in truncated_normal test by Eivind Jahren (#7726)
2586a4f007f97723cf3f947aee78c915029965ff Fix a lifetime issue of GUILogHandler by Eivind Jahren (#7730)
c093e12fadffb2617652b79eb73Ff762fc954cd72 Bump ruff to v0.4.0 by Lars Evje (#7707)
d6da82165aaa5444608e8ddeb6f7edba6ed2b3ac Make flakiness test for qstat faster by Håvard Berland (#7688)
57b4398186f3c28e62c75df37c8e6ff4e282f01e Reduce timeout in order to speed up tests by Håvard Berland (#7678)
21387c147a433e3099d0b7957cefaa142016b95d Allow pre-commit to run without failure in CI after merge by Håvard Berland (#7631)
dbafb2cb1fbfbadd974b43e75cddc0f750291a99 Add test to confirm that the job does not retry when activelly cancelled by Julius Parulek (#7625) 
61046df67fa516164388276c2483d89114c78fe8 Increase timeout for cluster integration tests by Håvard Berland (#7620)
880886ef8de3f8c066f862d1c478bb6f63bc9c0e Fix storage instance used outside of storage context by Jonathan Karlsen (#7614)
076b6d83342ff505f224e70a1e8ba9c0d18f54e0 Fix test_small_time_mismatches_are_ignored by Sondre Sortland (#7613)
6f49e28aa529c7d18de9513954ee848fa04d7b1b Protect main branch from git commit with pre-commit by Håvard Berland (#7585)
f4b0b2448f91a9907f329dd174d51ad37844657d Fix hanging cli integration test by Jonathan Karlsen (#7601)
cfad65e85d01a10fe8c57b4e391caefa2868d4dd Let mocked bkill support specific kill signal by Håvard Berland (#7599)
27a906029e3287634630fd0ff871dc395625ccd2 Increase timeout in unresponsiveness test by Håvard Berland (#7552)
b5a72e3e1572471b927d64c0146fc739283c900c Add --disable-monitor to Ert client runs in integration tests by Håvard Berland (#7552) 
414bea772d1f89c95fe82211d3e621e446950dc3 Fix storage instance used outside of storage context by Jonathan Karlsen (#7579)
7145a2924ebd5b9d7f051f42880f54cbfc1f4229 Test that pbs driver ignores qstat flakiness (#7414)
06bc54e706c15d3e45c8a6daeaf0c146ddaea70d Fix string arguments to driver.submit() by Håvard Berland (#7549)
cdc90ba3a1c267254d7bccdc957ffe290c2d379a Fix integration test lsf driver invalid resource requirement by Jonathan Karlsen (#7553)
9c51f1c14a34fc301fac86b6916baf74e04f1454 Ensure no underflow in parameter_example_test by Eivind Jahren (#7580)

### PR DESCRIPTION
**Pull request by eivindjahren**
**Issue**
Resolves #7770

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

**Review by oyvindeide — COMMENTED** [review]
Looks good, some smaller comments.

File: docs/about/release_notes.rst [hunk 1]
```diff
@@ -27,6 +27,55 @@
...
+------------
+
+Changes to the Manage experiments tool
+######################################
+
+The Manage experiments tool has become nicer to work with. Before you would just get text information about each experiment:
+![Screenshot from 2024-04-30 08-49-37](https://github.com/equinor/ert/assets/32731672/5db26f19-e772-4178-85ba-670b35a21e8b)
+.. image:: v9_manage_experiments.png
+
+Now there is a separate panel for viewing the information:
```
[the remark this answers is not in the document]
Comment (oyvindeide) [reply docs/about/release_notes.rst:40]: Or if you want to embellish:

>The interface for the Manage Experiments tool has been improved to provide more information about ensembles and experiments.
>
>A new panel has been introduced, which is dedicated to displaying detailed information for each experiment. This streamlines the process of reviewing and analyzing experiment data.
>

File: docs/about/release_notes.rst [hunk 2]
```diff
@@ -27,6 +27,55 @@
...
+
+Click on the square for the long running realization, 0 in the image:
+
+.. image:: click-on-realization.png
+
+Now click the "OPEN" button in the column STDERR for row ECLIPSE100:
+
+.. image:: click-on-stderr.png
+
+If the message contains "Eclipse failed due to license failure, retrying in XXs" then the license server is busy and we will automatically retry running eclipse in the specified number of seconds.
```
[the remark this answers is not in the document]
Comment (eivindjahren) [reply docs/about/release_notes.rst:66]: Done

**Review by andreas-el — APPROVED** [review]
Delicate work! 🎨

**Pull request merged**